graph/README.md

```diff
@@ -0,0 +1,33 @@
+### Instructions
+
+In order to run we first need to create a virtual environment.
+We recommend using [uv](https://docs.astral.sh/uv/getting-started/installation/)
+Once you have  `uv` installed you need to do
+```
+uv venv -p python3.10 graph
+source graph/bin/activate
+uv pip install matplotlib numpy pandas
+```
```

Comment:
.venv is idiomatic - we need to add to the .gitignore as well.
Also make a requirements file for installation of python requirements.